### ISSUE TITLE
Should we switch to `audio/x-m4a` for mp4 enclosures?

### ISSUE THREAD

**Issue by eteubert** [podlove/podlove-publisher#1180]
We currently mark feed enclosures for `.m4a` files as `audio/mp4` according to [iana](https://www.iana.org/assignments/media-types/audio/mp4) specifications.

However the [Apple Guidelines](https://help.apple.com/itc/podcasts_connect/#/itcb54353390) only list `audio/x-m4a` as a valid option (full list of accepted mime types: `audio/x-m4a, audio/mpeg, video/quicktime, video/mp4, video/x-m4v, and application/pdf`).

**Comment by Inrumpo** [comment]
Podlove is currently marking `.m4a` files as `type="audio/mp4"` within the `<enclosure>` tag.

**Comment by eteubert** [comment]
Thanks for the correction, fixed the typos 👍

**Comment by Inrumpo** [comment]
I did test this by going to `Podlove` → `Experten Einstellungen` → `Datei-Typen` and manually changing the MIME-Type for `MPEG-4 AAC Audio` to `audio/x-m4a`. This did change my `type` within my RSS feed to `type="audio/x-m4a"`.

I did not notice any other changes.
- The web player is working.
- Downloading an episode is working.
- The RSS feed does validate just the way it did before (Podbean Validator, Cast Feed Validator).
- Streaming via other platforms (player.fm, podchaser.com, Apple Podcasts Web, Google Podcasts Web, Pocket Casts Web) is working.

I then added the same MIME-Type to my web servers config – in case this might be needed. Still, nothing has changed. 

To sum this up: I did not see any changes, for better or worse.

**Comment by eteubert** [comment]
We will keep it as it is. Seems reasonable to use the iana standard rather than the proprietary one, as long as Apple doesn't enforce their spec.
